canonical-web-and-design/ubuntu.com
# Hide 'Featured in' section and update blog strip typography'

## Done

- Temporarily hid the 'featured in' section to move the blog posts up
- Updated the typography of the blog posts slightly including adding one more row in the line clamp

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the featured in section is not visible and the blog section is higher.


## Issue / Card

Fixes #8337

## Screenshots

![image](https://user-images.githubusercontent.com/441217/93893530-a25dfb80-fce5-11ea-9de8-bf3a4cb0f38b.png)
